packages/tools/replay-tool/src/helpers.ts

```diff
@@ -92,7 +101,10 @@ export async function loadContainer(
         new Map<string, IResolvedUrl>([[resolved.url, resolved]]),
     );
 
-    const dataStoreFactory = new ReplayDataStoreFactory(mixinDataStoreWithAnyChannel());
+    const dataStoreFactory = new ReplayDataStoreFactory(
+        strictChannels
+            ? undefined
```

Comment:
this is the strict channels support. currently only set true by the snapshot test. so the replay tool itself will continue to support unknown channels